extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -365,17 +365,23 @@ export class UpdateProjectFromDatabaseDialog {
 	private createProjectLocationRow(view: azdata.ModelView): azdata.FlexContainer {
 		const browseFolderButton: azdata.Component = this.createBrowseFileButton(view);
 
-		const value = this.project ? this.project.projectFilePath : '';
+		let values: string[] = [];
+		this.workspaceProjects.forEach(projectUri => {
+			values.push(projectUri.fsPath as string);
+		});
+
+		const value = this.project ? this.project.projectFilePath : (values[0] ? values[0] : '');
```

Comment:
`values[0] ? values[0] : ''` can be simplified to `values[0] ?? ''`

Reply:
Done.